client/src/views/client-area/ClientAreaPage.vue

```diff
@@ -203,10 +201,20 @@ onMounted(async () => {
     if (billingSystem === BillingSystem.CustomOrder || billingSystem === BillingSystem.ExperimentalCandidate) {
       currentPage.value = ClientAreaPages.Contracts;
       const statusResp = await BmsAccessInstance.get().getCustomOrderStatus(currentOrganization.value);
-      if (!statusResp.isError && statusResp.data && statusResp.data.type === DataType.CustomOrderStatus) {
-        if (statusResp.data.status === CustomOrderStatus.NothingLinked) {
+      const organizationStatusResp = await BmsAccessInstance.get().getOrganizationStatus(currentOrganization.value.bmsId);
+      const isBootstrapped =
+        !organizationStatusResp.isError && organizationStatusResp.data && organizationStatusResp.data.type === DataType.OrganizationStatus
+          ? organizationStatusResp.data.isBootstrapped
+          : false;
+      if (!organizationStatusResp.isError && organizationStatusResp.data) {
+        isBootstrapped;
+      }
+      if (!statusResp.isError && statusResp.data && statusResp.data.type === DataType.CustomOrderStatus && isBootstrapped) {
+        if (statusResp.data.status !== CustomOrderStatus.ContractEnded && !isBootstrapped) {
```

Comment:
You're only entering the first if `if isBootstrapped`, and now you're testing `if !isBootstrapped`. I think there's some confusion with all those conditions. Maybe adding some comments could help.